cmd/rcd/rcd.go

```diff
@@ -48,14 +48,23 @@ See the [rc documentation](/rc/) for more info on the rc flags.
 			rc.Opt.Files = args[0]
 		}
 
-		s, err := rcserver.Start(context.Background(), &rc.Opt)
+		ctx := context.Background()
+
+		s, err := rcserver.Start(ctx, &rc.Opt)
 		if err != nil {
 			log.Fatalf("Failed to start remote control: %v", err)
 		}
 		if s == nil {
 			log.Fatal("rc server not configured")
 		}
 
+		go func() {
```

Comment:
This means that the metrics only run for `rclone rcd`. Currently the metrics run whenever you have the `--rc` flag.

Reply:
Got it, I'll rework the behavior to the original one.